src/shared/parameter_v2/config_definitions/passing_config.yaml

```diff
@@ -0,0 +1,119 @@
+ - double:
+     name: static_field_position_quality_x_offset
+     min: 0
+     max: 1
+     value: 0.3
+     description: >-
+                The offset from the sides of the field to place the rectangular
+                sigmoid we use to determine what areas to pass to
+ - double:
+     name: static_field_position_quality_y_offset
+     min: 0
+     max: 1
+     value: 0.3
+     description: >-
+                The offset from the sides of the field to place the rectangular
+                sigmoid we use to determine what areas to pass to
+ - double:
+     name: static_field_position_quality_friendly_goal_distance_weight
+     min: 0
+     max: 1
+     value: 0.3
+     description: >-
+         The weight that being close to the goal will have on the static
+         position quality. Lower, more negative weights result in the distance
+         to the goal having less of an effect.
+ - double:
+     name: enemy_proximity_importance
+     min: 0
+     max: 10
+     value: 0.5
+     description: >-
+                This controls how heavily we weight an enemy robot being near the pass
+                receiver when calculating enemy risk to a pass
+ - double:
+     name: ideal_max_rotation_to_shoot_degrees
+     min: 0
+     max: 180
+     value: 60
+     description: >-
+                The maximum angle that we have to rotate after receiving a pass to
+                shoot that we think would likely result in a goal. Note that we may
+                try to take shots that require us to rotate more then this, it's more
+                of a soft limit.
+ - double:
+     name: min_pass_speed_m_per_s
```

Comment:
Some values in the `handheld_controller_config` such as `max_linear_speed` don't include units while others do. Is there a standard notation for if we want to include units in the names?

Reply:
This will be handled by this #1738 , we should have a standard to inlude units in the name or atleast in the description